addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -742,6 +742,7 @@ blockquote {
     overflow: hidden;
```

Comment:
Commit message:
- `parallax effects, and videos  were extended beyond rounded` => remove the double space after "videos" (also nitpick but there is no need of a comma after "effects") :wink: 
- > This commit adds the border-radius property to the background layer
placeholder and sets it to inherit, ensuring that all the options
mentioned above adhere to rounded borders.

  => we should not describe the code in the commit message (there is the diff to see that) but explain what is its effect  :wink:  I would reformulate it like this:
" This commit ensures that all these background options containers follow the rounded borders, by making them inherit the `border-radius` of their parent. "

- Maybe add some steps to reproduce :wink: